examples/camera/source/main.c

```diff
@@ -0,0 +1,196 @@
+#include <stdio.h>
```

Comment:
It's very likely that people will just copy this code and use it as it is right now. It may be worth moving all camera utilities to a different C file, and adding a copyright header to the C and H files so that they can include them in their projects.

Reply:
I'm not super confident that this is the best way to extract data from the camera yet. If it is, I'll probably move some of the code into libnds utility methods.

Reply:
Fair enough.

Reply:
I did, however, add licensing information.